internal/attachment/storage/filesystem/filesystem_attachment_test.go

```diff
@@ -0,0 +1,80 @@
+package filesystem_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	filestore "github.com/meroedu/meroedu/internal/attachment/storage/filesystem"
+	"github.com/meroedu/meroedu/internal/domain"
+	"github.com/meroedu/meroedu/pkg/log"
+)
+
+func createTempFile(filename string) (*os.File, error) {
```

Comment:
Should not return a `*os.File` here since it leads to a potential memory leak. It's will become a bad scenario if someone forgets to call `file.Close()`.

Reply:
yeah. I will change that.